compiler/rustc_middle/src/middle/limits.rs

```diff
@@ -1,12 +1,11 @@
 //! Registering limits:
-//! * recursion_limit,
-//! * move_size_limit, and
-//! * type_length_limit
+//! - recursion_limit
+//! - move_size_limit
+//! - type_length_limit
+//! - pattern_complexity_limit
 //!
-//! There are various parts of the compiler that must impose arbitrary limits
-//! on how deeply they recurse to prevent stack overflow. Users can override
```

Comment:
nit: I liked that part of the comment, it gives context; could you restore it?

Reply:
I removed it because it was just about `recursion_limit`, presumably from a time before the other limits were added. I will reinstate it in a way that makes it clear it's only describing `recursion_limit`.

Reply:
oh well, too late